roles/smartgateway/templates/metrics-configmap.yaml.j2

```diff
@@ -6,7 +7,12 @@ metadata:
 data:
   "smartgateway_config.json": |
     {
-      "AMQP1MetricURL": "{{ amqp_url | default('messaging-internal-' + meta.name + '.' + meta.namespace + '.svc:5672/collectd/telemetry') }}",
+      "AMQP1Connections": [
+        {
+          "URL": "{{ amqp_url | default('messaging-internal-' + meta.name + '.' + meta.namespace + '.svc:5672/collectd/telemetry') }}",
+          "DataSource": "{{ amqp_data_source | default('collectd') }}"
```

Comment:
No problem, but just want to draw attention to the fact that when the SG3 code merges, the collectd metrics container will no longer read this configmap at all. In that case, this default will be a little confusing. 

Is it worth an issue to come back to this and change it to ceilometer once the SG3 code merges? It will be weird to default to collectd in so many places, but to ceilometer here. Not sure exactly what to do, and it's minor, but thought I'd draw attention to this detail.

Reply:
I agree, let's open an issue and we can deal with this as part of the set of changes we need to handle as part of SG3 work. Once we merge ceilometer metrics work, then we can rebase the SG3 work on top of that, and address anything that seems off.

Reply:
(I have opened a new issue that refers to this comment)